earthly/docs/services/Readme.md

```diff
@@ -0,0 +1,19 @@
+# Documentation Service Definitions
+
+<!-- cspell: words livedocs sitedocs -->
+
+## Live Documentation
+
+To build documentation live, from the root of the repo run:
+
+```sh
+docker compose up -f local/docker-compose.livedocs.yml up --abort-on-container-exit
```

Comment:
Just a thought: in the past we'd typically use a Nix devshell in order to add a "task runner" to our repos. In other words, special commands that could be easily remembered and invoked to perform common operations. This isn't unique to Nix, though. We can start using an actual task runner to encapsulate some of these wordy invocations.